packages/backend/src/apps/delay/actions/delay-until/index.ts

```diff
@@ -31,9 +31,7 @@ export default defineAction({
     const { delayUntil, delayUntilTime } = $.step.parameters
     const delayTimestamp = generateTimestamp(
       delayUntil as string,
-      (delayUntilTime as string) === ''
-        ? defaultTime
-        : (delayUntilTime as string),
+      (delayUntilTime as string) || defaultTime,
```

Comment:
nit: would be goot to comment that we use `||` because we also want to catch empty string.